ddcommon/src/entity_id/unix/mod.rs

```diff
@@ -52,8 +56,18 @@ fn compute_entity_id(
         )
 }
 
+/// Set cgroup mount path to mock during tests
+/// # Safety
+/// Must not be called in multi-threaded contexts
+pub fn set_cgroup_file(path: String) {
```

Comment:
This question applies to all of the functions in the PR...Can we gate them behind a feature like `cgroup-test`? I know we are just adding back previous functionality, but I feel uneasy exposing functions intended for tests publicly like this. Forcing a consumer to opt-in is better than nothing.

Reply:
This is certainly not ideal. I was not aware that ddtrace php imported these directly...
The issue is that currently ddtrace php cannot compile their rust code in the repo because that function was removed.